float64[8] rbdl_gravity_compensation
float64[8] rbdl_pose_control
float64[8] rbdl_effort_control
float64[8] rbdl_total

float64[8] pin_gravity_compensation
float64[8] pin_pose_control
float64[8] pin_effort_control
float64[8] pin_total